setup.py

```diff
@@ -32,6 +32,7 @@
         'ruamel.yaml >=0.13.0, <0.16',
         'simplejson >=3.6, <4',
         'mido >=1.1, <2',
+        'tabulate >=0.8.9, <1',
```

Comment:
While SemVer says anything goes at the `0.x` stage, it's fairly common to treat the second digit as the major number in such cases. I suggest therefore limiting this to `<0.9` rather than `<1`.
```suggestion
        'tabulate >=0.8.9, <0.9',
```